core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.spec.ts

```diff
@@ -154,9 +159,29 @@ describe('Edit Thumbnail Modal Component', () => {
     component.onFileChanged(file);
 
     expect(component.uploadedImage).toBeNull();
+    expect(component.invalidFilenameWarningIsShown).toBeFalse();
     expect(component.invalidImageWarningIsShown).toBeTrue();
   });
 
+  it('should not load file if it does not have a valid filename', () => {
+    spyOn(component, 'isUploadedImageSvg').and.returnValue(true);
+    spyOn(component, 'isValidFilename').and.returnValue(false);
+    expect(component.invalidImageWarningIsShown).toBe(false);
+    expect(component.invalidFilenameWarningIsShown).toBe(false);
+    // This is just a mocked base 64 in order to test the FileReader event
+    // and its result property.
+    const dataBase64Mock = 'PHN2ZyB4bWxucz0iaHR0cDo';
+    const arrayBuffer = Uint8Array.from(
+      window.atob(dataBase64Mock), c => c.charCodeAt(0));
+    const file = new File([arrayBuffer], 'thumb..nail.svg');
```

Comment:
Can we add more file names to this test? Just to be sure we cover all of these cases: The svg file must not start with a dot or have '/' or 2 consecutive dots.

Reply:
Will do that